src/main/resources/sql/seed.sql

```diff
@@ -0,0 +1,90 @@
+-- Inserting data into dealerships
+INSERT INTO dealerships (name, address, phone)
```

Comment:
So yeah you are doing multiple dealerships, so you need to hook the repo into the Dealership class.